android-components/components/support/ktx/build.gradle

```diff
@@ -59,6 +60,7 @@ dependencies {
     androidTestImplementation Dependencies.androidx_test_runner
```

Comment:
out of scope for this, lines 47-49 are duplicates and need removing. Will create an issue for this